dynamodb.py

```diff
@@ -0,0 +1,219 @@
+"""
+awslimitchecker/services/dynamodb.py
+The latest version of this package is available at:
+<https://github.com/jantman/awslimitchecker>
+################################################################################
+Copyright 2015-2017 Jason Antman <jason@jasonantman.com>
+    This file is part of awslimitchecker, also known as awslimitchecker.
+    awslimitchecker is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+    awslimitchecker is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Affero General Public License for more details.
+    You should have received a copy of the GNU Affero General Public License
+    along with awslimitchecker.  If not, see <http://www.gnu.org/licenses/>.
+The Copyright and Authors attributions contained herein may not be removed or
+otherwise altered, except to add the Author attribution of a contributor to
+this work. (Additional Terms pursuant to Section 7b of the AGPL v3)
+################################################################################
+While not legally required, I sincerely request that anyone who finds
+bugs please submit them at <https://github.com/jantman/pydnstest> or
+to me via email, and that you send any contributions or improvements
+either as a pull request on GitHub, or to me via email.
+################################################################################
+AUTHORS:
+Jason Antman <jason@jasonantman.com> <http://www.jasonantman.com>
+################################################################################
+"""
+
+import abc  # noqa
+import logging
+
+from .base import _AwsService
+from ..limit import AwsLimit
+
+logger = logging.getLogger(__name__)
+
+
+class _DynamodbService(_AwsService):
+
+    service_name = 'DynamoDB'
+    api_name = 'dynamodb'  # AWS API name to connect to (boto3.client)
+
+    def find_usage(self):
+        """
+        Determine the current usage for each limit of this service,
+        and update corresponding Limit via
+        :py:meth:`~.AwsLimit._add_current_usage`.
+        """
+        
+        logger.debug("Checking usage for service %s", self.service_name)
+
+        self.connect()
+        self.connect_resource()
```

Comment:
I don't see ``self.resource_conn`` being used anywhere in this class; if we're not using the Resource-based access, we shouldn't be opening a resource connection.